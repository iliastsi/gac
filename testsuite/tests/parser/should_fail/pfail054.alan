ForwardDeclaration() : proc
  foo(x : int) : proc;
  foo2() : proc
  { foo(5); }
{}
