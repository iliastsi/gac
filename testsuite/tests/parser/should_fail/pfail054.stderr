pfail054.alan:2:22: Error: Parse error on input `;'
	Alan doesn't support function prototypes
	Use -XForwardDeclarations if you want to enable this

